docs/user/running-cincinnati.md

```diff
@@ -0,0 +1,91 @@
+# Running Cincinnati
+
+## Prerequisite
+
+APP-SRE publishes the Cincinnati container images at [Quay registry](https://quay.io/repository/app-sre/cincinnati)
+
+We need an image tag from the [Quay registry](https://quay.io/repository/app-sre/cincinnati) to use with the deployment config.
+
+The Cincinnati deploymentconfig present in the git repository at [dist/openshift/](../../dist/openshift/cincinnati.yaml)
+
+### Steps
+
+#### Create a new namespace for Cincinnati
+
+```shell
+oc new-project cincinnati
+```
+
+#### Create Cincinnati deployment
+
+```shell
+export IMAGE_TAG=<TAG>
+oc new-app -f cincinnati.yaml \
+  -p IMAGE_TAG=${IMAGE_TAG}
```

Comment:
Later in this doc, you just hard-code `90efacd` as the tag.  Maybe do that here too and drop the `IMAGE_TAG` shell variable?

Reply:
I went the other way as declaring the IMAGE_TAG environment variable is a better practice. So added that change in the script.